cmd/ttn-lw-cli/commands/end_devices.go

```diff
@@ -1083,6 +1107,7 @@ func init() {
 	endDevicesUpdateCommand.Flags().AddFlagSet(setEndDeviceFlags)
 	endDevicesUpdateCommand.Flags().AddFlagSet(attributesFlags())
 	endDevicesUpdateCommand.Flags().Bool("touch", false, "set in all registries even if no fields are specified")
+	endDevicesUpdateCommand.Flags().Bool("external-join-server", false, "disassociate and delete the device from cluster-local Join Server")
```

Comment:
The concept "cluster-local" does not exist in the CLI. If the CLI is configured to point to TTI's "global" join server, this flag would delete it from there.